doc/uninstall.md

```diff
@@ -0,0 +1,44 @@
+# Uninstalling the DevWorkspace Operator
+The DevWorkspace Operator makes use of finalizers on DevWorkspace resources and webhooks to restrict access to development resources. As these are not automatically removed when the Operator is uninstalled, they must manually be removed to fully clean the operator from the cluster.
+
+**Note:** The DevWorkspace Operator utilizes validating webhooks to restict `pods/exec` access to workspace resources. As Kubernetes provides no way to restrict which pods these webhooks apply to, improper removal of the webhook server could result in exec access to all pods on the cluster being blocked.
+
+1. Ensure that all DevWorkspace Custom Resources are removed along with their related k8s objects, like deployments.
+
+	```
+	kubectl delete devworkspaces.workspace.devfile.io --all-namespaces --all --wait
+	kubectl delete devworkspaceroutings.controller.devfile.io --all-namespaces --all --wait
```

Comment:
Seems devworkspace templates are missing. It's not a requirement to remove them since they don't have finalizers yet but still what is the point to leave them on the cluster?

Reply:
Good catch! I forgot we excluded them from WTO so they wouldn't be included in my copy-paste :)